src/libs/ReportUtils.ts

```diff
@@ -1247,54 +1332,52 @@ function getPersonalDetailsForAccountID(accountID) {
 
 /**
  * Get the displayName for a single report participant.
- *
- * @param {Number} accountID
- * @param {Boolean} [shouldUseShortForm]
- * @param {Boolean} shouldFallbackToHidden
- * @returns {String}
  */
-function getDisplayNameForParticipant(accountID, shouldUseShortForm = false, shouldFallbackToHidden = true) {
+function getDisplayNameForParticipant(accountID?: number, shouldUseShortForm = false, shouldFallbackToHidden = true): string | undefined {
     if (!accountID) {
         return '';
     }
     const personalDetails = getPersonalDetailsForAccountID(accountID);
     // this is to check if account is an invite/optimistically created one
     // and prevent from falling back to 'Hidden', so a correct value is shown
     // when searching for a new user
-    if (lodashGet(personalDetails, 'isOptimisticPersonalDetail') === true) {
+    if (personalDetails.isOptimisticPersonalDetail === true) {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Is there any specific reason for this?

Reply:
there is an eslint rule that is forcing using `??` operator but there are some cases where we cant use it , for example when value can be an empty string we need to use `||` operator. thats why sometimes we need to disable this rule